bulk_grades/views.py

```diff
@@ -32,11 +33,14 @@ def _configure(self, request, course_id):
             if self.processor.course_id != course_id:
                 return HttpResponseForbidden()
         else:
+            assignment_id = request.GET.get('assignment')
             self.processor = api.GradeCSVProcessor(
                                                   course_id=course_id,
                                                   _user=request.user,
                                                   track=request.GET.get('track'),
-                                                  cohort=request.GET.get('cohort')
+                                                  cohort=request.GET.get('cohort'),
+                                                  subsection=UsageKey.from_string(assignment_id) if assignment_id else None,
```

Comment:
Setting self.subsection to a usagekey may cause json serialization (for deferred processing) to fail. You can get around this by renaming it to `_subsection`.

Reply:
Y'know what, I fixed this on the wrong branch. It'll be corrected in https://github.com/edx/edx-bulk-grades/pull/15